src/master-playlist-controller.js

```diff
@@ -255,6 +260,11 @@ export class MasterPlaylistController extends videojs.EventTarget {
     // setup segment loaders
     // combined audio/video or just video when alternate audio track is selected
     this.mainSegmentLoader_ = new SegmentLoader(segmentLoaderOptions);
+
+    // for now we don't want the audio segment loader to also add cues to the segment
+    // metadata track
+    delete segmentLoaderOptions.segmentMetadataTrack;
```

Comment:
I think it would be better to create two different objects.

Perhaps using `Object.create` to make the `mainSegmentLoader_`'s object with `segmentMetadataTrack` added to the base object.